File: dataset/human/c/file_2350.c
#include<stdio.h>
#include<string.h>
#include<math.h>
 
 
int main(){
   int n,k,s;
   scanf("%d%d%d", &n,&k,&s);
   for(int i=0;i<n;i++){
     if(s<1000000000){
     	if(i<k){
     		printf("%d", s);
        }else {
            printf("%d", s+1);
        }
      } else{
         if(i<k){
     		printf("%d", s);
        }else {
            printf("1");
        }	 
      }
      if(i!=n-1){
         	printf(" "); 
      }
   }
	return 0;
}

File: dataset/human/python/file_2350.py
from math import ceil

d, g = map(int, input().strip().split())
points = [list(map(int, input().strip().split())) for _ in range(d)]

answer = float("inf")
for combination in range(2 ** d):
    score = 0
    solved = 0
    for difficuty in range(d):
        if (combination >> difficuty) & 1:
            score += points[difficuty][0] * 100 * (difficuty + 1) + points[difficuty][1]
            solved += points[difficuty][0]
    if score < g:
        for difficuty in range(d)[::-1]:
            if not (combination >> difficuty) & 1 and g - score < points[difficuty][0] * 100 * (difficuty + 1):
                solved += ceil((g - score) / (100 * (difficuty + 1)))
                break
        else:
            continue
    if answer > solved:
        answer = solved
print(answer)
